libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -269,8 +271,6 @@ void GCS_MAVLINK::send_battery_status(const uint8_t instance) const
     } else {
         consumed_wh = -1;
     }
-    uint8_t _percentage = -1;
-    const int8_t percentage = battery.capacity_remaining_pct(_percentage, instance) ? _percentage : -1;
```

Comment:
Here is what was suggested for this from the PR that last touched these lines. The changes just didn't make it into the PR.
https://github.com/ArduPilot/ardupilot/pull/13712#discussion_r698157230

```cpp
uint8_t _percentage = 0;
const int8_t percentage = battery.capacity_remaining_pct(percentage, instance) ? percentage : -1;
```